ansible/roles/fips/tasks/redhat-8.yaml

```diff
@@ -0,0 +1,18 @@
+- name: "redhat 8 fips: ensure fips_enabled is 1"
+  lineinfile:
+    dest: /proc/sys/crypto/fips_enabled
+    line: "1"
+    state: present
+  become: yes
+  check_mode: yes
+  register: sysfips
+
+- name: "redhat 8 fips: run fips enable command"
+  command: fips-mode-setup --enable
+  when:
+    - sysfips is changed
+
+- name: "redhat 8 fips: reboot to pick up fips changes"
+  ansible.builtin.reboot:
+  when:
+    - sysfips is changed
```

Comment:
nit: `sysfips is changed` status check is valid. wdyt adding another condition to check success status of `fips-mode-setup --enable` task

Reply:
please disregard above. the reboot task will proceed only if `fips-mode-setup --enable` is successful. no need to unnecessary check for its success.